_tools/epfl_infrastructure.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: EPFL Infrastructure
+category: tools
+order: 999
```

Comment:
? Why not just 3?

Reply:
expecting more "IC..." stuff. But anyway this should be changed if tools/middle are merged.